projects/packages/codesniffer/src/Utils/NamespaceInfo.php

```diff
@@ -176,7 +176,7 @@ public static function unqualifyClassName( $name, $nsname, array $aliases ) {
 			$unprefixed = null;
 		}
 		// Skip if the shortened version is an alias though.
-		if ( isset( $aliases[ $unprefixed ] ) ) {
+		if ( isset( $aliases[ $unprefixed ?? '' ] ) ) {
```

Comment:
I think for this I'd prefer an explicit null check.
```suggestion
		if ( $unprefixed !== null && isset( $aliases[ $unprefixed ] ) ) {
```

Reply:
I had that on my list to go back and clean up, as I didn't like it either. 😄